services/src/test/java/edu/unc/lib/dl/cdr/services/rest/modify/ExportCsvIT.java

```diff
@@ -164,9 +167,9 @@ public void exportCollectionCsv() throws Exception {
         List<CSVRecord> csvList = parseCsvResponse(response);
         assertEquals("Unexpected number of results", 2, csvList.size());
         assertContainerRecord(csvList, ResourceType.Collection, collObj.getPid(), "Collection",
-                COLLECTION_PATH, 2, false, 1, false);
+                COLLECTION_PATH, 2, false, 1, false, "Authenticated", false);
         assertContainerRecord(csvList, ResourceType.Folder, folderObj.getPid(), "Folder",
-                FOLDER_PATH, 3, false, null, false);
+                FOLDER_PATH, 3, false, null, false, "Restricted", false);
```

Comment:
Seems like the folder should have the same permissions as the parent

Reply:
Same as above comment